tgui/packages/tgui/interfaces/HumanInventory/index.tsx

```diff
@@ -59,13 +87,26 @@ type SlotProps = { name: string; slot: HumanInventorySlot };
 const Slot = (props: SlotProps, context) => {
   const { act } = useBackend<HumanInventoryData>(context);
   const { slot, name } = props;
-  const { id, item } = slot;
+  const { id, item, obstructed } = slot;
+
+  let getItemName = function getItemName(obstructed, item) {
+    if (obstructed) {
+      return 'Obstructed';
+    }
+    if (!item) {
+      return 'Nothing';
+    }
+    return item;
+  };
```

Comment:
Can pull this outside of the `Slot` component, as it only uses things passed to it. No need to define once per `Slot`.